docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -233,13 +233,61 @@ Check the full [release announcement](https://camel.apache.org/blog/2021/06/came
 +
   * Click the green "Publish release" button at the bottom
 
+== Upgrade and tag Examples
+
+* Make sure all https://github.com/apache/camel-quarkus-examples/pulls[PRs] against `camel-quarkus-main` branch are merged.
+* Upgrade to the Camel Quarkus version you just released
++
+[source,shell]
+----
+$ NEW_CQ_VERSION=... # E.g. 2.2.0
+$ git fetch upstream
+$ git checkout camel-quarkus-main
+$ git reset --hard upstream/camel-quarkus-main
+$ find . -type f -name pom.xml -exec sed -i "s|<camel-quarkus.version>[^<]*</camel-quarkus.version>|<camel-quarkus.version>$NEW_CQ_VERSION</camel-quarkus.version>|g" {} \;
+$ ./mvnw-for-each.sh org.l2x6.cq:cq-maven-plugin:0.25.0:sync-example-properties
+$ git commit -m "Upgrade to Camel Quarkus $NEW_CQ_VERSION"
+----
++
+* Make sure that the tests are still passing:
++
+[source,shell]
+----
+$ ./mvnw-for-each.sh clean verify -Pnative
+----
++
+* If everything works for you locally, open a PR to merge `camel-quarkus-main` to `main`
+* Once the PR is merged, tag the `main` branch with the `$NEW_CQ_VERSION`:
++
+[source,shell]
+----
+$ git checkout main
+$ git fetch upstream
+$ git reset --hard upstream/main
+$ ./mvnw-for-each.sh versions:set -DnewVersion=$NEW_CQ_VERSION
```

Comment:
I don't catch something. Are we setting the versions a second time ? Feel like it was already done on camel-quarkus-main and then merged ?

Reply:
Yes, here we set the `project.version` s of the individual examples. Above, we have set the value of `camel-quarkus.version` property.

Reply:
Indeed, a good sleep after, it's clearer now :)